EU4toV2/Source/EU4World/EU4Country.cpp

```diff
@@ -105,6 +105,8 @@ EU4::Country::Country(
 	possibleShogun(false),
 	militaryLeaders(),
 	government(),
+	governmentRank(),
```

Comment:
Another bad example from the surrounding code, but by adding default values in the header, it becomes unnecessary to initialize them here.

Reply:
dropped initialization here.